src/AccessibilityInsights/Modes/TestModeControl.xaml.cs

```diff
@@ -261,8 +262,13 @@ public async Task SetElement(Guid ecId)
                 // Improves the reliability of the rendering but does not solve across the board issues seen with all WPF apps.
                 this.Dispatcher.BeginInvoke(DispatcherPriority.Send, new Action(() =>
                 {
-                    Application.Current.MainWindow.InvalidateVisual();
-                    Application.Current.MainWindow.Visibility = Visibility.Visible;
+                    // This is a workaround for a repaint issue caused by an interaction between Jaws and WPF
+                    if (NativeMethods.IsExternalScreenReaderActive())
```

Comment:
This is returning true for me after I close JAWS. Do we know if the value needs a log out or reboot to update?

Reply:
This is mainly a nicety to reduce unnecessary flashing. If we assume that most people never install Jaws, the "lingering" status shouldn't be a problem.

Reply:
Fair enough!